lib/ansible/modules/utilities/helper/_accelerate.py

```diff
@@ -16,11 +16,14 @@
 DOCUMENTATION = '''
 ---
 module: accelerate
-removed: True
```

Comment:
I'm not sure there is good reason to keep this any longer.  We have versioned docs now, so we should just `git rm` this.

Reply:
bcoca said this file is complex and we don't want to remove it yet.